lib/con_cache.ex

```diff
@@ -79,10 +79,10 @@ defmodule ConCache do
   Starts the server and creates an ETS table.
 
   Options:
-    - `{:ttl_check, time_ms}` - A check interval for TTL expiry. This value is
+    - `{:ttl_check_interval, time_ms}` - A check interval for TTL expiry. This value is
```

Comment:
`time_ms | false`

The description should also be adapted. There is no default value (it must be supplied), and we should explain the `false` option.